src/applications/personalization/profile-2/components/MilitaryInformation.jsx

```diff
@@ -36,35 +129,7 @@ const MilitaryInformationContent = ({ militaryInformation, veteranStatus }) => {
     invalidVeteranStatus &&
     !militaryInformation?.serviceHistory?.serviceHistory
   ) {
-    return (
-      <AlertBox
-        isVisible
-        status="warning"
-        headline="We don't seem to have your military records"
-        content={
-          <>
-            <p>
-              We're sorry. We can't match your information to our records. If
-              you think this is an error, please call the VA.gov help desk at{' '}
-              <Telephone contact={CONTACTS.HELP_DESK} /> (TTY:{' '}
-              <Telephone contact={CONTACTS['711']} pattern={PATTERNS['911']} />
-              ). We’re here Monday–Friday, 8:00 a.m.–8:00 p.m. ET.
-            </p>
-            <p>
-              Or you can learn how to{' '}
-              <a
-                href="https://www.archives.gov/veterans/military-service-records/correct-service-records.html"
-                target="blank"
-                rel="noopener noreferrer"
-              >
-                update or correct your military service history
-              </a>
-              .
-            </p>
-          </>
-        }
-      />
-    );
+    return <NotAVeteranAlert />;
```

Comment:
I am confused - Are you sure that the new copy is still preferred? I added this new copy only last sprint. It is possible that we throw a not in MVI error because of a server error, and thus the user should not be prompted to take action.

Reply:
For reference, it was this PR https://github.com/department-of-veterans-affairs/vets-website/pull/14236

Reply:
I'm also confused. I just moved some JSX into a component, but didn't actually change any alerts in this file.

Reply:
Sorry, I mixed up the nonVeteran vs. MPI alert. Must have had a case of Friday brains 🧠

Reply:
🧟 🧠